synapse/models/entity.py

```diff
@@ -122,6 +122,10 @@
                 'doc': 'An interface for entities which have initiative to act.'}),
 
             ('entity:singular', {
+                'interfaces': (
+                    ('geo:locatable', {'prefix': 'birth:place', 'template': {'happened': 'was born'}}),
```

Comment:
could use a `title` template value

Reply:
This interface doesn't define a title on purpose. Anything which implements the interface does.